sqlserver/datadog_checks/sqlserver/sqlserver.py

```diff
@@ -134,15 +135,19 @@ def initialize_connection(self):
         except Exception as e:
             self.log.exception("Initialization exception %s", e)
 
-    def handle_service_check(self, status, host, database, message=None):
+    def handle_service_check(self, status, host, database, message=None, is_default=True):
         custom_tags = self.instance.get("tags", [])
         if custom_tags is None:
             custom_tags = []
+
         service_check_tags = ['host:{}'.format(host), 'db:{}'.format(database)]
         service_check_tags.extend(custom_tags)
         service_check_tags = list(set(service_check_tags))
 
-        self.service_check(SERVICE_CHECK_NAME, status, tags=service_check_tags, message=message, raw=True)
+        if self.autodiscovery:
```

Comment:
Looks like there's a merge conflict with the master branch here

Reply:
thanks for pointing it out!